TS04-Plotting-Time-Series-R.Rmd

```diff
@@ -100,12 +103,12 @@ harMet15.09.11$datetime<-as.POSIXct(harMet15.09.11$datetime,
                     format = "%Y-%m-%dT%H:%M",
                     tz = "America/New_York")
 #daily HARV met data, 2009-2011
-harMetDaily.09.11 <- read.csv(file="AtmosData/HARV/hf001-06-daily-m.csv",
+harMetDaily.09.11 <- read.csv(file="Met_HARV_Daily_2009_2011.csv",
                      stringsAsFactors = FALSE)
 
 #covert date to Date type
 
-harMetDaily.09.11$date<-as.Date(harMetDaily.09.11$date)
+harMetDaily.09.11$date <- as.POSIXct(harMetDaily.09.11$date)
```

Comment:
@mguarinello When dealing with date only and not time.  Why use POSIXct instead of Date?  Both were taught in these lessons (granted as.Date is in 00, not this one).